test/unit/io/structure_file/format_vienna_test.cpp

```diff
@@ -236,75 +226,68 @@ struct read_fields : public read {};
 TEST_F(read_fields, only_seq)
 {
     std::stringstream istream{input};
-    for (size_t idx = 0ul; idx < expected_seq.size(); ++idx)
+    structure_file_input fin{istream, format_vienna{}, fields<field::SEQ>{}};
+    auto it = fin.begin();
+    for (size_t idx = 0ul; idx < expected_seq.size(); ++idx, ++it)
     {
-        EXPECT_NO_THROW(( format.read(istream, options, seq, ig, ig, ig, ig, ig, ig, ig, ig) ));
-
-        EXPECT_TRUE(std::ranges::equal(seq, expected_seq[idx]));
-        seq.clear();
+        EXPECT_TRUE(std::ranges::equal(get<field::SEQ>(*it), expected_seq[idx]));
     }
 }
 
 TEST_F(read_fields, only_id)
 {
     std::stringstream istream{input};
```

Comment:
include sstream

Reply:
it is included :)

Reply:
Oh true, in this file the includes were visually hidden in Github's folding mechanism, so I didn't spot them...